src/App.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {Component} from 'react';
+import Navbar from './components/Navbar/Navbar';
+import UserBoard from './components/UserInput/UserBoard';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null
+    }
+  }
+
+  fetchData = () => {
+    axios.get("https://poetrydb.org/random")
```

Comment:
nice job using the poetry api. there are a ton of free apis that you could play around with to get more practice with this